src/ast/query.rs

```diff
@@ -152,6 +153,30 @@ impl fmt::Display for SetQuantifier {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct Table {
```

Comment:
```suggestion
/// A [`TABLE` command]( https://www.postgresql.org/docs/current/sql-select.html#SQL-TABLE)
pub struct Table {
```

Reply:
I had to look up that this indeed is part of postgres syntax: https://www.postgresql.org/docs/current/sql-select.html#SQL-TABLE